src/scenes/Login/Login.js

```diff
@@ -1,18 +1,23 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import './_login.scss'
 import PropTypes from 'prop-types'
+import { getToken, saveToken } from '../../useToken'
 
-const Login = ({ history }) => {
+const Login = () => {
   const [username, setUsername] = useState()
   const [password, setPassword] = useState()
   const [error, setError] = useState(null)
-
   const handleChangUsername = e => {
     setUsername(e.target.value)
   }
   const handleChangePassword = e => {
     setPassword(e.target.value)
   }
+  useEffect(() => {
+    if (getToken()) {
+      return window.location.replace('/')
+    }
+  }, [])
```

Comment:
For a better structure, should move this section below the function `handleSubmit` and use a blank line to separate them
Good structure should be:
- Variables declaration section
- Function declaration section
- hook section

Reply:
And no need to `return`

Reply:
![image](https://user-images.githubusercontent.com/62316127/125916571-5caa1fb6-2800-410b-9649-ee3c1d239e1a.png)